site/gatsby-site/src/components/discover/Filters.js

```diff
@@ -2,10 +2,26 @@ import React from 'react';
 import REFINEMENT_LISTS from 'components/discover/REFINEMENT_LISTS';
 import Filter from './Filter';
 import { useMenuContext } from 'contexts/MenuContext';
+import { graphql, useStaticQuery } from 'gatsby';
 
 function Filters() {
   const { isCollapsed } = useMenuContext();
 
+  const {
+    taxa: { nodes: taxa },
+  } = useStaticQuery(graphql`
+    query FiltersTaxaQuery {
+      taxa: allMongodbAiidprodTaxa(filter: { namespace: { in: ["CSETv1", "CSETv0", "GMF"] } }) {
```

Comment:
for later: we should identify all the places where taxa are hard-coded and move it into the config (as started in this PR).

Reply:
Sadly, that's a limitation of Gatsby's `useStaticQuery.` [You can't do dynamic queries](https://www.gatsbyjs.com/docs/how-to/querying-data/use-static-query/#differences-between-page-queries--static-queries).

We should refactor that one eventually to use a page query or dynamically pull from the graphql endpoint.